src/contexts/browserReducer.js

```diff
@@ -0,0 +1,28 @@
+const browserReducer = (state, action) => {
```

Comment:
you can destructure type and payload, over here so that you can directly use payload without the need of action

Reply:
Cool, will update it too